app/controllers/application_controller.rb

```diff
@@ -2,8 +2,16 @@
 
 class ApplicationController < ActionController::API
   include DeviseTokenAuth::Concerns::SetUserByToken
+  include Pundit
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :authenticate_user!
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  private
+
+  def user_not_authorized
+    render json: { error: 'You are not authorized to perform this action.' }
```

Comment:
The HTTP has already defined [codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status) for some typical situations. In current shape, the server returns `200` (its the default value) which means "The request has succeeded." which is a lie. The `render` has a possibility to set the HTTP code for a response by `status` parameter. Take a look at the description in [the Rails guide](https://guides.rubyonrails.org/v2.3/layouts_and_rendering.html#the-status-option), please.

For me, the https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403 seems to be the most appropriate in the case.

Reply:
Makes sense, thanks 👍  I set the status as forbidden.